src/compiler/tsbuild.ts

```diff
@@ -809,6 +810,7 @@ namespace ts {
                     // *after* those files, then we're "psuedo up to date" and eligible for a fast rebuild

```

Comment:
sp:pseudo

Reply:
I need a VS Code Extension just for this word